app/javascript/pages/Greeting.js

```diff
@@ -6,16 +6,16 @@ const Greeting = () => {
   const dispatch = useDispatch();
   const greeting = useSelector((state) => state.greetings);
 
+
   useEffect(() => {
     dispatch(fetchGreeting());
   }, []);
 
   return (
-    <div>{Object.values(greeting).map((greeting) => (
-      <h1>{greeting.message}</h1>
-  ))}
-  </div>
-    )
+    greeting?.map(greet => {
```

Comment:
[OPTIONAL] You completed the previous reviewer's requirement well. However, your app should generate random greetings and display them in the front end when we refresh the page.

Reply:
@Rustamxon7  can you give morer insights  on this optional requiremennt? where was this made and i couln't find a link to your project to even see how you did yours. i am waiting to hear back from you

Reply:
Hi @Efiamotu-1, I will leave the student's project link. [The link](https://github.com/Camilovelag/hello-rails-react/blob/dev/app/controllers/api/v1/greetings_controller.rb).